exonum/src/api/private/system.rs

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::net::SocketAddr;
-use std::collections::HashMap;
-
+use serde_json;
 use router::Router;
 use iron::prelude::*;
 
+use std::net::SocketAddr;
+use std::collections::HashMap;
```

Comment:
Not sure there's a real rule for  it, but it seems that std imports are usually first?

Reply:
Nope: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions
:smile: